packages/client/hmi-client/src/components/models/tera-model-diagram.vue

```diff
@@ -73,7 +73,7 @@
 			<!-- Model equations -->
 			<AccordionTab header="Model equations">
 				<section :class="isEditingEQ ? `diagram-container-editing` : `diagram-container`">
-					<section v-if="props.isEditable" class="controls">
+					<section v-if="props.isEditable && props.isEquationsEditable" class="controls">
 						<Button
```

Comment:
the `isEditable` prop seems to encompass a wide variety of editability within the componenet.  We need a specific prop for when just equations themselves can be edited

Reply:
Food for thoughts for @shawnyama